File: 2043.py
password, input = map(int, input().split())
number_of_times = 0

while password - input >= 0 :
    number_of_times += 1
    input += 1

print(number_of_times)


# first_number, second_number = map(int, input().split())

# print(first_number + second_number)
# print(first_number - second_number)
# print(first_number * second_number)
# print(first_number // second_number)

File: 2021년/SW Expert Academy/2043.c
#include <stdio.h>
#pragma warning (disable:4996)

main()
{
	int password, input, number_of_times = 0;
	
	printf("Enter a password and a number :");
	scanf("%d %d", &password, &input);

	while (password - input >= 0)
	{
		++number_of_times;
		++input;
	}
	printf("%d", number_of_times);
}